starlette/middleware/wsgi.py

```diff
@@ -62,7 +62,7 @@ def build_environ(scope: Scope, body: bytes) -> dict:
 
 
 class WSGIMiddleware:
-    def __init__(self, app: typing.Callable) -> None:
+    def __init__(self, app: typing.Callable[..., typing.Any]) -> None:
```

Comment:
Went for `Callable[..., Any]` as this is module is deprecated. Let me know if you want to have the complete definition.

Reply:
That makes sense, let's not spend time typing this.